buildmodel/commands.go

```diff
@@ -190,6 +190,17 @@ func SubmitUpdatePR(f *PRFlags) error {
 		Purpose: "auto-update",
 	}
 
+	// if using github app we need to insert the app token into the URL
+	// so that the app can push to the repo
+	if *f.Auth.GitHubAppClientID != "" {
```

Comment:
Why the `if`? Isn't this the case for PATs too, now that it isn't being injected into `origin`?

Reply:
agreed, removed the if